### ISSUE TITLE
KeyError: 'messages', no other error from the command line

### ISSUE THREAD

**Issue by scmanjarrez** [acheong08/EdgeGPT#123]
Hi, 
I've been getting the following error:

```python                                                                                                                                                                                      
        EdgeGPT - A demo of reverse engineering the Bing GPT chatbot                                                                                                                          
        Repo: github.com/acheong08/EdgeGPT                                                                                                                                                    
        By: Antonio Cheong                                                                                                                                                                    
                                                                                                                                                                                              
        !help for help

        Type !exit to exit
        Enter twice to send message or set --enter-once to send one line message
     
Initializing...

You:
hello world

Bot:
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/capitano/telegrambots/Edge-GPT-Telegram-Bot/venv/lib/python3.8/site-packages/EdgeGPT.py", line 439, in <module>
    asyncio.run(main())
  File "/usr/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/capitano/telegrambots/Edge-GPT-Telegram-Bot/venv/lib/python3.8/site-packages/EdgeGPT.py", line 396, in main
    async for final, response in bot.ask_stream(
  File "/home/capitano/telegrambots/Edge-GPT-Telegram-Bot/venv/lib/python3.8/site-packages/EdgeGPT.py", line 316, in ask_stream
    async for response in self.chat_hub.ask_stream(
  File "/home/capitano/telegrambots/Edge-GPT-Telegram-Bot/venv/lib/python3.8/site-packages/EdgeGPT.py", line 263, in ask_stream
    yield False, response["arguments"][0]["messages"][0][
KeyError: 'messages'
```

I used the cli trying to debug the error, but no other information is being returned. Any idea what could be the cause? I'm using EdgeGPT 0.0.56.1
Edit: I've re-exported cookies, still happening. The cookies seem fine, at least I can talk with the chat using Microsoft Edge

**Comment by jiayuqi7813** [comment]
i fix that，maybe u can try this #124

**Comment by tom68-ll** [comment]
I meet this error too.

**Comment by scmanjarrez** [comment]
Closes as fixed on #124
